packages/block-editor/src/components/spacing-sizes-control/style.scss

```diff
@@ -82,8 +95,8 @@
 
 	.components-spacing-sizes-control__range-control {
 		grid-column: span 3;
-		padding-right: $grid-unit-10;
-		height: 30px;
+		height: 40px;
```

Comment:
Should we use `$grid-unit-50` here? Unsure how the other control updates have specified their height.

Reply:
At the moment [color indicator](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/color-indicator/style.scss#L2) is the only component using that var, and it does a `*0.5` on it to get `20px`.

Reply:
Perhaps it doesn't matter too much at this point. The Spacing Range component may need to be refactored a bit to align with other controls? cc @ciampo @mirka

Reply:
Let's _not_ use `$grid-unit-50` — it's not something that we've been using much in `@wordpress/components` (color indicator is a legacy component and in this case a bit of an exception)